content/en/serverless/installation/python.md

```diff
@@ -93,16 +93,16 @@ Transform:
     Parameters:
       stackName: !Ref "AWS::StackName"
       apiKey: <DATADOG_API_KEY>
-      nodeLayerVersion: "<LAYER_VERSION>"
+      pythonLayerVersion: "<LAYER_VERSION>"
       extensionLayerVersion: "<EXTENSION_VERSION>"
       service: "<SERVICE>" # Optional
       env: "<ENV>" # Optional
 ```
 
 To fill in the placeholders:
 - Replace `<DATADOG_API_KEY>` with your Datadog API key from the [API Management page][4]. 
-- Replace `<LAYER_VERSION>` with the desired version of the Datadog Lambda layer (see the [latest releases][5]).
-- Replace `<EXTENSION_VERSION>` with the desired version of the Datadog Lambda Extension (see the [latest releases][6]).
+- Replace `<LAYER_VERSION>` with the desired version of the Datadog Lambda layer. The latest version is `{{< latest-lambda-layer-version layer="python" >}}`.
+- Replace `<EXTENSION_VERSION>` with the desired version of the Datadog Lambda Extension. The latest version is `{{< latest-lambda-layer-version layer="extension" >}}`.
```

Comment:
Could we just directly replace the values in the code snippet with the latest layer/extension versions?

Reply:
Yes, we could.

Advantage: no need to copy-paste the value.
Disadvantage: less obvious that you can choose a different value.

What do you think?

Reply:
Let's make it easy to copy paste, but maybe keep a note that the latest Extension version has been pre-populated and will need to be updated in the future